### PR TITLE
[fix bug 1419120] Implement State of Mozilla 2016 pages

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Adds State of Mozilla 2016 report at `/foundation/annualreport/2016/` behind switch `som-2016`.
- There is still one outstanding question over an image/chart that needs to be resolved, so do not merge for now but this is good for review.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1419120

## Testing
- Check for typo's?
- Test responsiveness.

### CODE REVIEW

File: bedrock/foundation/templates/foundation/annualreport/2016/index.html [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+            <p>
+            {% trans cliqz='https://blog.mozilla.org/press-uk/2016/08/23/mozilla-makes-strategic-investment-in-cliqz-to-enable-privacy-focused-search-innovation/' %}
+              The majority of Mozilla Corporation’s revenue is from royalties earned through Firefox
+              web browser search partnerships and distribution deals around the world. Mozilla
+              Corporation’s revenue and income support for CY 2016 was $506M, as compared to $414M in
+              CY 2015. Over the past several years, Mozilla has shifted from one having global search
+              default in Firefox to a more local and flexible approach by country to offer users more
+              choice and to enable innovation and competition in search. In 2015 and 2016, we expanded
+              this new global search strategy and secured new partnerships with DuckDuckGo, Google and
+              Yahoo <sup>1</sup>. In 2016, Mozilla Corporation began making strategic investments and acquisitions
```
Comment (jpetto) [inline bedrock/foundation/templates/foundation/annualreport/2016/index.html]: Should we link this`<sup>1</sup>` to the footnote?

File: bedrock/foundation/templates/foundation/annualreport/2016/index.html [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+              advancing the Mozilla mission and protecting internet health, which means that all of our
+              resources at Mozilla contribute to the overall sustainability and health of the internet.
+            {% endtrans %}
+            </p>
+
+            <p>
+            {% trans download=url('firefox.new'), volunteer=url('mozorg.contribute.index'), mozorg=url('mozorg.home') %}
+              Our mission is to ensure the internet is a global public resource, open and accessible to all.
+              This mission is as important now as it has ever been. We urge everyone who is interested to
+              find a way to contribute to protecting the internet. We invite you to <a href="{{ download }}">download</a> Firefox or
```
Comment (jpetto) [inline bedrock/foundation/templates/foundation/annualreport/2016/index.html]: Maybe make "download Firefox" the link?

File: bedrock/foundation/templates/foundation/annualreport/2016/index.html [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+            {% trans %}
+              The State of Mozilla 2016 is our annual report. This report highlights
+              activities for 2016 and is accompanied by detailed financials. This report
+              is released when we submit the Mozilla non-profit tax filing for the previous
+              calendar year.
+            {% endtrans %}
+            </em></p>
+
+            <p>
+                {% trans %}
```
Comment (jpetto) [inline bedrock/foundation/templates/foundation/annualreport/2016/index.html]: Nit - indentation.

**Comment by jpetto** [comment]
A blast from the past creeps in on mobile/RTL:

screen shot 2017-11-30 at 11 54 38 am

**Comment by alexgibson** [comment]
Thanks @jpetto - I pushed a commit with some review fixes. 

I'm not going to delay pushing this change any longer, so removing the DNM label. If they want design tweaks, we can follow up later.

**Comment by alexgibson** [comment]
Squashed!

**Pull request merged**